examples/src/main/scala/org/apache/spark/examples/SparkKMeans.scala

```diff
@@ -33,7 +33,6 @@ object SparkKMeans {
   def parseVector(line: String): Vector[Double] = {
     DenseVector(line.split(' ').map(_.toDouble))
   }
-
```

Comment:
Actually, I guess this newline does not have to be removed because it seems, in 
 [scala-style-guide#blank-lines-vertical-whitespace](https://github.com/databricks/scala-style-guide#blank-lines-vertical-whitespace),
>A single blank line appears:
>Between consecutive members (or initializers) of a class: fields, constructors, methods, nested classes, static initializers, instance initializers